dbf2csv/models/dbf2csv.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
```

Comment:
Transient models / views should go into the `wizards/` directory, as per [OCA guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories).

Reply:
Ok, thanks

Reply:
Ok, thanks